Kysect.BotFramework/Core/BotMessages/BotImageAndVideoMessage.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Kysect.BotFramework.ApiProviders;
+
+namespace Kysect.BotFramework.Core.BotMessages
+{
+    public class BotImageAndVideoMessage : IBotMessage
+    {
+        public string Text { get; }
+        public string ImagePath { get; }
+
+        public string VideoPath { get; }
+        
+        public BotImageAndVideoMessage(string text, string imagePath, string videoPath)
+        {
+            Text = text;
+            ImagePath = imagePath;
+            VideoPath = videoPath;
+        }
+        
+        public void Send(IBotApiProvider apiProvider, SenderInfo sender)
+        {
+            apiProvider.SendImage(ImagePath, Text, sender);
+            apiProvider.SendVideo(VideoPath, "", sender);
```

Comment:
Какая-то рандомная константа "" , почему бы не сделать вот так
```suggestion
            apiProvider.SendVideo(VideoPath, sender);
```